src/components/withKeyboardState.tsx

```diff
@@ -34,14 +41,9 @@ function KeyboardStateProvider(props) {
     return <KeyboardStateContext.Provider value={{isKeyboardShown}}>{children}</KeyboardStateContext.Provider>;
 }
 
-KeyboardStateProvider.propTypes = keyboardStateProviderPropTypes;
-
-/**
- * @param {React.Component} WrappedComponent
- * @returns {React.Component}
- */
-export default function withKeyboardState(WrappedComponent) {
-    const WithKeyboardState = forwardRef((props, ref) => (
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export default function withKeyboardState(WrappedComponent: ComponentType<{ref: ForwardedRef<unknown>}>) {
```

Comment:
Add return type for this function, similar how Bartek did [here](https://github.com/Expensify/App/pull/29426/files)